package-lock.json

```diff
@@ -465,9 +465,9 @@
       "dev": true
     },
     "cacache": {
-      "version": "9.3.0",
-      "resolved": "https://registry.npmjs.org/cacache/-/cacache-9.3.0.tgz",
-      "integrity": "sha512-Vbi8J1XfC8v+FbQ6QkOtKXsHpPnB0i9uMeYFJoj40EbdOsEqWB3DPpNjfsnYBkqOPYA8UvrqH6FZPpBP0zdN7g==",
+      "version": "10.0.1",
```

Comment:
Do we absolutely need all of these upgrades in this PR? I get that it's a good idea, but it would be especially annoying to deal with potential problems during this release

Reply:
I wanted to update the version of the github markdown css; these subdep changes came from updating our top level dependencies with bugfixes.